Containers/nextcloud/start.sh

```diff
@@ -50,6 +50,17 @@ if ! [ -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file" ]; then
 fi
 sudo -u www-data rm -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file"
 
+# Correctly set CPU_ARCH for notify_push
+CPU_ARCH="$(uname -m)"
+export CPU_ARCH
+if [ -z "$CPU_ARCH" ]; then
+    echo "Could not get processor architecture. Exiting."
+    exit 1
+elif [ "$CPU_ARCH" != "x86_64" ]; then
+    export CPU_ARCH="aarch64"
+    ADDITIONAL_APKS+=" gcompat"
```

Comment:
@Zoey2936 do you have [recognize](https://github.com/nextcloud/recognize) running on arm64? If yes, was gcompat for it required to work?

Reply:
I run AIO only on amd64 and there it works without gcompat, so no Idea if it is required on aarch64

Reply:
Ah I see, thanks nonetheless!